dev/tools/dartdoc.dart

```diff
@@ -50,12 +50,7 @@ Future<void> main(List<String> arguments) async {
   final StringBuffer buf = StringBuffer();
   buf.writeln('name: Flutter');
   buf.writeln('homepage: https://flutter.dev');
-  // TODO(dnfield): We should make DartDoc able to avoid emitting this. If we
-  // use the real value here, every file will get marked as new instead of only
-  // files that have otherwise changed. Instead, we replace it dynamically using
-  // JavaScript so that fewer files get marked as changed.
-  // https://github.com/dart-lang/dartdoc/issues/1982
-  buf.writeln('version: 0.0.0');
+  buf.writeln('version: 0.0.0'); // Change this?
```

Comment:
@dnfield can you weigh in on what should change here now that this issue has been resolved?

Reply:
https://github.com/dart-lang/dartdoc/pull/2010/files added the ability to exclude the version in the footer info. I can no longer remember if it makes more sense to still do it this way or not - we would still want to inject the actual version number at runtime, but if we can make that dartdoc option work with that logic we can drop the ogic here that is forcing the version to initially appear as `0.0.0`.

This is all being done to prevent firebase from seeing us changing _every single page_ of the docs on every single commit.